### DISCUSSION TITLE
Theme folder on Docker?

### DISCUSSION THREAD

**Discussion by satonotdead** [navidrome/navidrome#2713] [Q&A]
I'm trying to mount the theme folder and enter docker to manage the theme.js file but I don't know where are the files :)

What's the suggested way to change themes using Docker? Thanks!

**Comment by caiocotts** [comment]
Currently, if you'd like to add more themes to Navidrome, you'll need to change the source code. There ins't a folder that you can just drop themes into and load them.

You may want to refer to Navidrome's documentation on creating new themes:
https://www.navidrome.org/docs/developers/creating-themes/
